app/src/main/java/com/mapbox/maps/testapp/examples/IconPropertyActivity.kt

```diff
@@ -0,0 +1,113 @@
+package com.mapbox.maps.testapp.examples
+
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.expressions.generated.Expression
+import com.mapbox.maps.extension.style.image.image
+import com.mapbox.maps.extension.style.layers.generated.symbolLayer
+import com.mapbox.maps.extension.style.layers.properties.generated.IconAnchor
+import com.mapbox.maps.extension.style.sources.generated.vectorSource
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_add_marker_symbol.*
+
+/**
+ * Example displaying multiple images in a symbol layer
+ */
+class IconPropertyActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_icon_property)
+
+    mapView.getMapboxMap().loadStyle(
+      styleExtension = style(Style.OUTDOORS) {
+        // Add icons from the U.S. National Parks Service to the map's style.
+        +image(RESTROOMS) {
+          bitmap(BitmapFactory.decodeResource(resources, R.drawable.nps_restrooms))
+        }
+        +image(TRAIL_HEAD) {
+          bitmap(BitmapFactory.decodeResource(resources, R.drawable.nps_trailhead))
+        }
+        +image(PICNIC_AREA) {
+          bitmap(BitmapFactory.decodeResource(resources, R.drawable.nps_picnic_area))
+        }
+        // Access a vector tileset that contains places of interest at Yosemite National Park.
+        // This tileset was created by uploading NPS shapefiles to Mapbox Studio.
+        +vectorSource(SOURCE_ID) {
+          url(SOURCE_URI)
+        }
+        // Create a symbol layer and access the layer contained.
+        +symbolLayer(LAYER_ID, SOURCE_ID) {
```

Comment:
missing SourceLayer property here:
```suggestion
        +symbolLayer(LAYER_ID, SOURCE_ID) {
          // Access the layer that contains the Point of Interest (POI) data.
          // The source layer property is a unique identifier for a layer within a vector tile source.
          sourceLayer("Yosemite_POI-38jhes")
```